infra/config-store/src/main/java/com/simprints/infra/config/store/local/ConfigLocalDataSourceImpl.kt

```diff
@@ -179,7 +179,12 @@ internal class ConfigLocalDataSourceImpl @Inject constructor(
                     frequency = SynchronizationConfiguration.Frequency.PERIODICALLY,
                     up = UpSynchronizationConfiguration(
                         simprints = UpSynchronizationConfiguration.SimprintsUpSynchronizationConfiguration(
-                            kind = UpSynchronizationConfiguration.UpSynchronizationKind.NONE
+                            kind = UpSynchronizationConfiguration.UpSynchronizationKind.NONE,
+                            batchSizes = UpSynchronizationConfiguration.UpSyncBatchSizes(
+                                sessions = 1,
```

Comment:
Why isn't `UpSyncBatchSizes.default()` being used here and in the `OldProjectConfig` below?

Reply:
🤷🏻